index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My portfolio</title>
+</head>
+<body>
+    <header class="container">
+        <nav class="container-logo">
+            <a href="#">
+                <img src="images/logo.png" alt="logo">
+            </a>
+            <a href="#">
+                <img src="images/Union.png" alt="union" class="union">
+            </a>
+        </nav>
+        
+        <div class="main-container">
+            <div class="main-text">
+                <h2>I’m MeeMaw <br> Glad to see you!</h2>
+            </div>
+            <div class="text">
+                <h3>I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+                experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h3>
```

Comment:
- [ ] Also kindly use a `<p>` tag instead of a `<h3>` tag for the paragraph. This way will be semantically correct and the screen readers and other technologies will thank you